README.md

```diff
@@ -427,8 +427,8 @@ be installed with `cargo install cargo-make`).
 
 ## Apps
 
-Check out the list of more than 50 [Apps using
-`Ratatui`](https://github.com/ratatui-org/ratatui/wiki/Apps-using-Ratatui)!
+Checkout [awesome-ratatui](https://github.com/ratatui-org/awesome-ratatui) for a curated list of
```

Comment:
One last comment, I don't know which one is grammatically correct :D

```suggestion
Check out [awesome-ratatui](https://github.com/ratatui-org/awesome-ratatui) for a curated list of
```

Reply:
You're right, typo from me.

If I'm not mistaken "check out" is basically "to look at" while "checkout" is usually going to the cashier or when leaving an hotel (I don't know why git checkout is named so)